flow360/component/simulation/operating_condition.py

```diff
@@ -4,47 +4,24 @@
 
 from flow360.component.simulation.framework.base_model import Flow360BaseModel
 
-"""
-    Defines all the operating conditions for different physics.
-    The type of operating condition has to match its volume type.
-    Operating conditions defines:
-    1. The physical (non-geometrical) reference values for the problem.
-    2. The initial condition for the problem.
-
-    TODO:
-    1. What other types of operation conditions do we need?
-"""
-
-
-class TurbulenceQuantities(Flow360BaseModel):
-    """PLACE HOLDER, Should be exactly the the same as `TurbulenceQuantitiesType` in current Flow360Params"""
-
-    pass
 
+class DensityTemperature(Flow360BaseModel):
+    temperature: TemperatureType = pd.Field(288.15)
+    density: DensityType = pd.Field(1.225)
 
-class ExternalFlowOperatingConditions(Flow360BaseModel):
-    Mach: float = pd.Field()
-    alpha: float = pd.Field(0)
-    beta: float = pd.Field(0)
-    temperature: float = pd.Field(288.15)
-    reference_velocity: Optional[float] = pd.Field()  # See U_{ref} definition in our documentation
 
-    initial_flow_condition: Optional[tuple[str, str, str, str, str]] = pd.Field(
-        ("NotImplemented", "NotImplemented", "NotImplemented")
-    )
-    turbulence_quantities = Optional[TurbulenceQuantities] = pd.Field()
+class StandardAtmosphere(Flow360BaseModel):
+    altitude: LengthType = pd.Field(0)
+    temperature_offset: TemperatureType = pd.Field(0)
 
 
-class InternalFlowOperatingConditions(Flow360BaseModel):
-    pressure_difference: float = pd.Field()
-    reference_velocity: float = pd.Field()
-    inlet_velocity: float = pd.Field()
-
-
-class SolidOperatingConditions(Flow360BaseModel):
-    initial_temperature: float = pd.Field()
-
+class OperatingCondition(Flow360BaseModel):
+    """
+    Defines all the operating conditions for different physics.
+    The type of operating condition has to match its volume type.
+    Operating conditions defines the physical (non-geometrical) reference values for the problem.
+    """
 
-OperatingConditionTypes = Union[
-    ExternalFlowOperatingConditions, InternalFlowOperatingConditions, SolidOperatingConditions
-]
+    velocity: Optional[VelocityType] = pd.Field(None)
+    Mach: Optional[float] = pd.Field(None)
```

Comment:
the primary motivation here is user experience with interface. This is why we introduced operating conditions. They should contain alpha beta angles as well, as this is intended to be the only required part of simulation. Everything else can have defaults. This is problem centric approach. Specifying angles in freestream is solver centric approach. We just wanted to simplify it. To address your questions/points
1. I don't think Mach will be often provided. All our customers think in dimensioned values. We can come up with different input method if that is awkward
2. It is not meant to store info for nondimensionalization because user doesn't care about that. This is solver technical detail. It is side effect of operating conditions, not it's purpose.
3. ok